packages/cloud-cognitive/src/components/Saving/Saving.js

```diff
@@ -18,89 +18,90 @@ import PropTypes from 'prop-types';
 import { pkg } from '../../settings';
 const componentName = 'Saving';
 
-export let Saving = ({
-  cancelButtonText,
-  className,
-  defaultIconDescription,
-  defaultText,
-  failIconDescription,
-  failText,
-  inProgressIconDescription,
-  inProgressText,
-  onCancel,
-  onSave,
-  status,
-  successIconDescription,
-  successText,
-  type,
-}) => {
-  const getStatusObj = () => {
-    const statusObj = {};
+export let Saving = forwardRef(
+  (
+    {
+      secondaryButtonText,
+      className,
+      defaultIconDescription,
+      defaultText,
+      failIconDescription,
+      failText,
+      inProgressIconDescription,
+      inProgressText,
+      onRequestCancel,
+      onRequestSave,
+      status,
+      successIconDescription,
+      successText,
+      type,
+      ...rest
+    },
+    ref
+  ) => {
+    const statusObj = {
+      default: {
+        text: defaultText,
+        iconDescription: defaultIconDescription,
+        icon: CheckmarkOutline16,
+      },
+      inProgress: {
```

Comment:
Most strings/props are kebab case instead of camel case. `in-progress` also matches a few of our other components. Let's update this.